addons/web/static/src/views/calendar/calendar_renderer.scss

```diff
@@ -82,6 +82,10 @@
             border-color: transparent;
             border-bottom-color: $border-color;
             padding: 3px 0 5px;
+
+            .fc-week-number {
```

Comment:
This section of the code is called "Adapt calendar table borders".
You padding rules are not the only ones unrelated to borders in here, I think that we should rephrase the section title